### DISCUSSION TITLE
SBAS analysis issues

### DISCUSSION THREAD

**Discussion by Congzhangg** [gmtsar/gmtsar#510] [Q&A] [answered]
Hello, everyone. My research area is a mountain. The deformation from August 2017 to August 2020 was obtained using SBAS-InSAR in GMTSAR. When I compared InSAR displacement with GNSS data displacement, I found that there was a big difference between them. In the same time period, InSAR results are rising and GNSS results are falling. Please see the following figure for details.
 Which steps might lead to such results? Since InSAR results will be used later, is there any way to improve my InSAR results?

Question 2:, GMTSAR treatment of SBAS would get such a different result, or is the study area a mountain cause?
![image](https://user-images.githubusercontent.com/112800803/198044869-47d3f4bc-fd24-4d48-88c2-693a0ed8346d.png)

**Comment by Xiaohua-Eric-Xu** [comment, marked as answer]
Coherence does have an effect on time-series results. A few procedures to try here 1) stronger smoothness; 2) make redundant interferograms (e.g. 90-day temporal baseline) to suppress errors from decorrelation (note the sbas in GMTSAR is a coherence weighted sbas); 3) reference to a GNSS station or at least reference to a quiet are (with good quality). There are other artifacts that you could consider too, e.g. atmospheric noise, unwrapping errors, plate motion, etc.

**Reply by Congzhangg** [reply]
By the third method, do you mean to correct the interferogram using a GNSS data?

**Reply by Xiaohua-Eric-Xu** [reply]
Not necessarily, did you pin your interferogram to a position in your interferogram?

**Reply by Congzhangg** [reply]
No ，my study area is very small, only in part of a strip

**Reply by Xiaohua-Eric-Xu** [reply]
It does not depend on the size of your area, with our a reference point, there maybe random unwrapping ambiguities that'll affect your time-series. InSAR is still more of a relative measurement (unless one can correct every error source).

**Reply by Congzhangg** [reply]
OK，There are 32 images in my study. The temporal and spatial baselines are set to 300 200, and 220 interference pairs are obtained. The smoothness index is set to 8 during SBAS analysis, but the results are still not very good.  
![image](https://user-images.githubusercontent.com/112800803/198563419-8f4f5298-903a-44cc-ae56-2414bcb16f4f.png)

Is the baseline_line.dat file used in constructing select_pairs.csh the result when prepproc_batch_tops.csh mode is 1 or mode is 2? I'm using mode 1 result.

**Reply by Xiaohua-Eric-Xu** [reply]
Like I mentioned, you need to reference your data before putting it to sbas analysis. Otherwise, the solution has no physical meaning.

**Reply by omarwater98** [reply]
> La coherencia tiene un efecto en los resultados de las series de tiempo. Algunos procedimientos para probar aquí 1) mayor suavidad; 2) hacer interferogramas redundantes (p. ej., línea de base temporal de 90 días) para suprimir errores de descorrelación (tenga en cuenta que el sbas en GMTSAR es un sbas ponderado por coherencia); 3) referencia a una estación GNSS o al menos referencia a un área tranquila (con buena calidad). Hay otros artefactos que también podría considerar, por ejemplo, el ruido atmosférico, los errores de desenvolvimiento, el movimiento de las placas, etc.

So, to have the least decorrelation, would you have to propose a baseline that is as low as possible? Let's say 400 m and 400 days is a lot? and develop with the coherence indicated neither very good nor very poor?

The same thing happens to me. I have enough GNSS stations to compare but they vary a lot. Should I add more images and change the coherence with which it unfolds?

**Reply by Xiaohua-Eric-Xu** [reply]
@omarwater98 I am not sure I fully understand your questions. Usually more redundancy means robustness.

**Reply by omarwater98** [reply]
> @omarwater98 I am not sure I fully understand your questions. Usually more redundancy means robustness.

![Validacion con GNSS](https://user-images.githubusercontent.com/113547446/205336014-29f89245-882f-440e-94b1-c00c80f3253f.png)
i have this result of vel_ll.grd, and 32 GNSS points, but any of them match with the vel_ll.grd.
i used 23 images since 2014 for the InSAR process until 2022, my question is: If i use more images could i have more correlation (for the baseline maybe) and this will give me more precision on the result?

![Valor del pixel](https://user-images.githubusercontent.com/113547446/205337743-3e7e4e4c-f1ab-463e-8859-c87512d62d19.png)

or is maybe i have to make the unwrap with more coherence?, i used 0.01 (good coherence) maybe .05?

**Reply by Xiaohua-Eric-Xu** [reply]
> i used 23 images since 2014 for the InSAR process until 2022, my question is: If i use more images could i have more correlation (for the baseline maybe) and this will give me more precision on the result?

two questions here 1) why you only have 23 images between 14-22? Sentinel-1 typically have 6-24 day repeat. 2) more usually mean better but not for all cases, and InSAR is usually a relative measurement, thus you need to reference your data (e.g. to a station) before making comparisons.

**Reply by omarwater98** [reply]
I used 23 images because I selected the dates where there were no clouds, I did this by reviewing the lansat 8 images from USGS. Should I try with more images and more pairs again?

Referencing means you compare offsets to an existing station like this?

![01 Comparacion de estacion](https://user-images.githubusercontent.com/113547446/205720235-13b8065d-726c-495a-b730-b38b678d8b92.png)

**Reply by Xiaohua-Eric-Xu** [reply]
One of the major advantages of SAR measurement is that the microwave could penetrate through clouds. I don't think you should exclude any acquisitions in terms of doing a time series.
